### PR TITLE
Test 3 with Page Objects for checkout is added

### PR DESCRIPTION
**Pull request by Byelenka**

### CODE REVIEW

File: src/pages/CheckoutStepOne.page.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    checkoutZip = this.page.locator('[id="postal-code"]');
+
+    checkoutContinueButton = this.page.locator('[id="continue"]');
+
+    /**
+     * @param {string} firstName
+     * @param {string} lastName
+     * @param {string} zip
+     */
+    async fillYourInformation(firstName, lastName, zip) {
```
Comment (ihorBohdvn) [inline src/pages/CheckoutStepOne.page.js:22]: Add a check that after filling in the fields, the continue button becomes active, or vice versa, if any field is empty, an error appears

File: src/pages/CheckoutStepTwo.page.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        const sum = arrayOfPrices.reduce((a, b) => a + b);
+        return roundTo(sum * 1.08, 2);
+    }
+
+    async getTotalPrice() {
+        const totalPrice = await this.checkoutTotal.textContent();
+        // @ts-ignore
+        return parseFloat(totalPrice.match(/[\d,]+\.\d{2}/));
+    }
+}
```
Comment (ihorBohdvn) [inline src/pages/CheckoutStepTwo.page.js:58]: Add a check that the sum of all products matches the displayed total on the page

File: src/pages/ShoppingCart.page.js [hunk 3]
```diff
@@ -18,6 +18,8 @@ export class ShoppingCartPage extends BaseSwagLabPage {
 
     cartItems = this.page.locator(this.cartItemSelector);
 
+    cartCheckoutButton = this.page.locator('[id="checkout"]');
+
```
Comment (ihorBohdvn) [inline src/pages/ShoppingCart.page.js:22]: Add a check after deleting a product to make sure the product no longer exists

File: src/tests/testsCheckout.spec.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        expect(checkoutItemDescriptions).toEqual(itemDescriptions);
+
+        const checkoutItemPrices = await app.checkoutStepTwo.getProductPrices();
+        expect(checkoutItemPrices).toEqual(itemPrices);
+
+        const totalPrice = await app.checkoutStepTwo.calculateTotalPrice();
+        const price = await app.checkoutStepTwo.getTotalPrice();
+        expect(price).toEqual(totalPrice);
+    });
+});
```
Comment (ihorBohdvn) [inline src/tests/testsCheckout.spec.js:75]: Add tests that check what happens when incorrect data is entered or no data is entered

**Comment by ihorBohdvn** [comment]
Overall, a very good job! ))))You're doing great, keep it up! You just need to do a little tuning)

File: src/pages/ShoppingCart.page.js (hunk 3 above)
Comment (Byelenka) [reply src/pages/ShoppingCart.page.js:22]: Added here: src/tests/testsShoppingCart.spec.js

File: src/tests/testsCheckout.spec.js (hunk 4 above)
Comment (Byelenka) [reply src/tests/testsCheckout.spec.js:75]: To be honest, I don‘t know where to add this checks. For checkout-step-one I‘ve added checks for each field. And they accepts any symbols (even ±!@ and spaces). Checkout-page-two doesn't have any fields. I can only click Finish or Cancel there

File: .gitignore [hunk 5]
```diff
@@ -2,3 +2,7 @@ node_modules/
 /test-results/
 /playwright-report/
 /playwright/.cache/
+/test-results/
```
Comment (EugenePetrik) [inline .gitignore]: Please, remove duplicates

File: src/pages/CheckoutStepOne.page.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
+// @ts-check
+import { BaseSwagLabPage } from './BaseSwagLab.page';
+
+export class CheckoutStepOnePage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    checkoutFirstName = this.page.locator('[id="first-name"]');
```
Comment (EugenePetrik) [inline src/pages/CheckoutStepOne.page.js]: Or `#first-name`

Comment (EugenePetrik) [reply src/pages/CheckoutStepOne.page.js]: The same below

File: src/tests/testsCheckout.spec.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+import { users } from '../test-data/users';
+import { usersInfo } from '../test-data/usersInfo';
+
+const { username, password } = users.standardUser;
+const { firstName, lastName, zip } = usersInfo.standardUserInfo;
+
+test.beforeEach(async (
```
Comment (EugenePetrik) [inline src/tests/testsCheckout.spec.js]: Please, move the `beforeEach` hook to describe block

File: src/tests/testsCheckout.spec.js [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+        const itemNames = items.map((item) => item.name);
+        const itemDescriptions = items.map((item) => item.description);
+        const itemPrices = items.map((item) => item.price);
+
+        await app.inventory.shoppingCart.click();
+
+        await app.shoppingCart.cartCheckoutButton.click();
+        await expect(app.checkoutStepOne.checkoutError).toBeHidden();
+
+        await app.checkoutStepOne.checkoutContinueButton.click();
```
Comment (EugenePetrik) [inline src/tests/testsCheckout.spec.js]: Please create a separate method in page object to click on the button

Comment (EugenePetrik) [reply src/tests/testsCheckout.spec.js]: The same below

File: src/tests/testsShoppingCart.spec.js [hunk 9]
```diff
@@ -30,5 +30,10 @@ test.describe('Tests for Shopping card', () => {
 
         const cartItemPrices = await app.shoppingCart.getProductPrices();
         expect(cartItemPrices).toEqual(itemPrices);
+
+        await app.shoppingCart.removeCartItemById(0);
```
Comment (EugenePetrik) [inline src/tests/testsShoppingCart.spec.js:46]: It would be better to add a separate test for this verification

**Review by EugenePetrik — CHANGES_REQUESTED** [review]
Good job! Left a few comments

**Review by batmanastasiya — APPROVED** [review]
Good job!

**Pull request merged**